src/business/AccountService.java

```diff
@@ -0,0 +1,26 @@
+package business;
+
+
+import dataaccess.AccountRepository;
+
+
+public class AccountService {
+	AccountRepository acountRepo = new AccountRepository();
+	public boolean validateUser(String username, String password) throws LibrarySystemException {
+		
+		User userFromRepo = acountRepo.getUser(username, password);
+		if(userFromRepo == null)
+			return false;		
+		return true;
+	}
+
+	
+	public boolean CreateMember(LibraryMember member) throws LibrarySystemException {	
```

Comment:
argument for the method shouldn't be of type `LibraryMember` it should be the required fields needed to instantiate a `LibraryMember` object... this object should be the one passed to the repository layer